### PR TITLE
Update/trustless api

### PR DESCRIPTION
**Pull request by christopherbrumm**
This PR updates the Trustless API section with its latest changes.

### CODE REVIEW

File: docs/developers/data_engineers/accessing_data/trustless_api/how_to_use.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+:::tip
+This section describes how to use the Trustless API and how to access the endpoint by KYVE.
+:::
+
+## Endpoint
+The official endpoint of the Trustless API by the KYVE team can be found here: 
+
+**https://data.services.kyve.network**
+
+The endpoint provides data of various pools. For example, it includes the Ethereum // Blobs Pool from Testnet and the Celestia // Blobs Pool from Devnet.
```
Comment (mbreithecker) [inline docs/developers/data_engineers/accessing_data/trustless_api/how_to_use.md]: Maybe make clear that you are talking about KYVE's Testnet and Devnet pools. Maybe somebody thinks, that we only have Etherums Testnet data?

File: docs/developers/data_engineers/accessing_data/trustless_api/implementation.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+ 	-  [precomputes the Trustless API response](#precompute-trustless-api-response),
+  	-  [saves the precomputed response](#save-response--keys),
+  	-  and saves the response location for certain keys.
+3. Repeats that every n-seconds
+
+### Query Bundles
+
+To insert a bundle, the following steps are required for the crawler:
+
+1. Query the specific bundle using the bundleId on the KYVE chain, where it is called finalizedBundle (the ChildCrawler will use the chainrest defined in the config as the node endpoint).
```
Comment (mbreithecker) [inline docs/developers/data_engineers/accessing_data/trustless_api/implementation.md]: chainrest?

File: docs/developers/data_engineers/accessing_data/trustless_api/implementation.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+2. Retrieve the decompressed bundle data associated with the finalizedBundle from the storage provider (the ChildCrawler will use the storagerest defined in the config as the storage provider endpoint).
```
Comment (mbreithecker) [inline docs/developers/data_engineers/accessing_data/trustless_api/implementation.md]: guess chainrest and storagerest are variables? Probably put them in quotes

File: docs/developers/data_engineers/accessing_data/trustless_api/implementation.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+
+The purpose of the indexer is to return the possible indices for a specific data item, which are then stored and later queried in the database.
+
+**Example: `EthBlobs`**
+
+The `EthBlobsIndexer` generates all necessary indices to query for blobs:
+- block_height
+- slot_number
+
+This means, the `EthBlobsIndexer` will take a bundle, which is an array of data items, as an argument and return an array of trustless data items back. A trustless data item contains the actual data, the inclusion proof and all necessary information to verify that proof (like `chainId` and `bundleId`). Additionally it contains an array of indicies for that specific data item, these indicies will then be stored in the data base to correctly retrieve the trustless data item later on.
```
Comment (mbreithecker) [inline docs/developers/data_engineers/accessing_data/trustless_api/implementation.md]: "data base" -> "database"

**Review by mbreithecker — COMMENTED** [review]
Very nice!

File: docs/developers/data_engineers/accessing_data/trustless_api/implementation.md (hunk 2 above)
Comment (christopherbrumm) [reply docs/developers/data_engineers/accessing_data/trustless_api/implementation.md]: Defined in the config, will mark it as argument

**Review by mbreithecker — APPROVED** [review]

**Pull request merged**